modules/ROOT/pages/db/db-connector-insert-update-delete.adoc

```diff
@@ -3,48 +3,157 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The connector also provides insert, update and delete operations. Just like the xref:db/db-connector-query.adoc[select operation], these operations support dynamism and parameterization by using embedded DataWeave transformations:
+The Database connector supports insert, update, and delete operations.
+Similar to the SELECT operation, these operations support dynamic queries and
```

Comment:
From what I can tell, SELECT is not technically a DDL operation and is a DQL (data query language) operation along with SHOW and HELP. That said, I refer to other operations textually instead of as their syntax needs, like create rather than CREATE. I think I will make select lowercase here.